core-4/src/main/java/parkinglot/AppConfig.java

```diff
@@ -53,25 +54,12 @@ public Engine defaultV6Engine(){
     }
 
     @Bean("4X4")
-    public Car _4X4(@Qualifier("defaultV6Engine") Engine engine){
-        return new FordBronco(engine);
-    }
-
-    @Bean
-    public Car jurassicCar(@Qualifier("defaultV6Engine") Engine engine){
-        return new FordRaptor(engine);
+    public Car _4X4(){
+        return new FordBronco(defaultV6Engine());
     }
 
     @Bean
-    public Map<String, Car> carMap(@Qualifier("wheelsOfDevil") Car wheelsOfDevil,
-                                   @Qualifier("bumblebee") Car bumblebee,
-                                   @Qualifier("4X4") Car _4X4,
-                                   @Qualifier("jurassicCar") Car jurassicCar) {
-        return new HashMap<String, Car>() {{
-            put("wheelsOfDevil", wheelsOfDevil);
-            put("bumblebee", bumblebee);
-            put("4X4", _4X4);
-            put("jurassicCar", jurassicCar);
-        }};
+    public Car jurassicCar(){
+        return new FordRaptor(defaultV6Engine());
```

Comment:
nope, it's prohibited too, even in the previous task, please, put Engine back to the signature, but don't use any annotations, Spring should understand how to inject defaultV6engine to both cars